build

```diff
@@ -2,18 +2,27 @@
 
 set -e
 
-# Populate feature files like feed changes by reading thorugh the code and hooking up
-# all the calls we need, as well as processing settings files
-python populateFeaturesFiles.py
+# Generate the feedChanges file using all the legacy feature folders
+python generateFeedChanges.py
 
-# Check our code style with jscs.
-echo "[   INFO] Checking code style with ESLint..."
-npm run eslint --silent || exit
+# Generate legacy and new settings
+npm run gen-settings
+
+# Generate features/index.js for the new features
+npm run gen-featureIndex
+
+# Check legacy code with eslint
+echo "[   INFO] Checking legacy code style with ESLint..."
+npm run legacy-eslint --silent || exit
```

Comment:
webpack runs lint on all of the new files this runs lint on the legacy files. I figure you just run `./build` when you're deploying?